impeller/renderer/backend/vulkan/command_pool_vk.cc

```diff
@@ -14,6 +14,10 @@
 
 namespace impeller {
 
+// This number comes from observing the recycled_buffers_ size while running
+// Wonderous and seeing it cap out at 7.
+static constexpr size_t kMaxRecycledBufferSize = 10;
```

Comment:
This may be moot since they will still probably live in the pool.

Reply:
I'm not sure this is necessary since like you mention, its the pool itself that has memory. reseting/freeing should be more or less the same right?

Reply:
Yea, `kMaxRecycledBufferSize` isn't going to really work because it seems like the pool should be holding onto the command buffers anyways.  I removed it.